src/tools/plan-generator.ts

```diff
@@ -134,17 +136,16 @@ export class PlanGenerator {
   }
 
   /** Generates a consistent handle id. */
-  createCreateHandleId(handle: Handle): string {
+  async createCreateHandleId(handle: Handle): Promise<string> {
     if (handle.id) return handle.id;
     if (!this.createHandleRegistry.has(handle)) {
-      const rand = Math.floor(Random.next() * Math.pow(2, 50));
-      this.createHandleRegistry.set(handle, `handle/${rand}`);
+      this.createHandleRegistry.set(handle, await digest(this.randomSeed + this.createHandleIndex++));
```

Comment:
I'm rusty on this... is it a seed or a salt?

Reply:
Uses I know:
You give a seed to the random number generator.
You give salt before hashing a password, so that you cannot reconstruct or compare hashes easily.

Which one is this? Both? Neither? Maybe more like a salt, I will change to salt then.